### PR TITLE
Release using autopub

### PR DESCRIPTION
**Pull request by marcoacierno**
Test here: https://github.com/marcoacierno/strawberry/

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

New deployment process to release new Strawberry releases

File: pyproject.toml [hunk 1]
```diff
@@ -104,3 +104,9 @@ testpaths = ["tests/"]
 [build-system]
 requires = ["poetry>=0.12"]
 build-backend = "poetry.masonry.api"
+
+[tool.autopub]
+git-username = "Botberry"
+git-email = "bot@strawberry.rocks"
+project-name = "🍓"
+append-github-contributor = true
```
Comment (patrick91) [inline pyproject.toml:112]: Nice!

**Review by patrick91 — APPROVED** [review]
Nice one! @justinmayer will be happy to see we finally started using autopub 😊

**Comment by patrick91** [comment]
@marcoacierno feel free to merge this whenever you want :)

**Comment by justinmayer** [comment]
I am indeed thrilled to see this project's releases automated via AutoPub, which wouldn't exist without the two of you! ❤️

**Pull request merged**